sdk/contentsafety/ai-content-safety-rest/README.md

```diff
@@ -332,9 +356,9 @@ const client = ContentSafetyClient(endpoint, credential);
 const blocklistName = "TestBlocklist";
 const blockItemText1 = "sample";
 const blockItemText2 = "text";
-const addBlockItemsParameters: AddBlockItemsParameters = {
+const addOrUpdateBlocklistItemsParameters: AddOrUpdateBlocklistItemsParameters = {
   body: {
-    blockItems: [
+    blocklistItems: [
```

Comment:
same here for rename ?

Reply:
Yes, it also causes by our API contract changes.
![image](https://github.com/Azure/azure-sdk-for-js/assets/20239605/0649f293-0394-437d-b7d1-cf1e3fe35c31)